### PR TITLE
95 destination list

### PR DESCRIPTION
**Pull request by ItsTristan**
#95 

Finishes the destination list fragment. Includes a bunch of stuff from #18 (in progress).

It's basically just expense item list fragment copy-pasted.

### CODE REVIEW

File: ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/AddDestinationActivity.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import ca.ualberta.cs.shinyexpensetracker.models.Destination;
+import ca.ualberta.cs.shinyexpensetracker.models.ExpenseClaim;
+
+public class AddDestinationActivity extends Activity {
```
Comment (adamjford) [inline ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/AddDestinationActivity.java]: Need a Javadoc comment here.

File: ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/DestinationList.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package ca.ualberta.cs.shinyexpensetracker.models;
+
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DestinationList extends Model<DestinationList> {
```
Comment (adamjford) [inline ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/DestinationList.java]: If `ExpenseClaim` just has an `ArrayList<Destination>` to hold its child destinations, what is this class for?

**Comment by adamjford** [comment]
Needs to have `master` merged in.

File: ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/DestinationList.java (hunk 2 above)
Comment (ItsTristan) [inline ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/DestinationList.java]: That stuff was pulled from Ramish's branch (#18), which I believe he was in the middle of fixing when I pulled.

**Pull request merged**